### PR TITLE
Styling of media library modal

### PR DESCRIPTION
**Pull request by loumray**
Tried to be consistent with the theme.  Hope this help. Give it a try and let me know how it looks for you.

Btw, thanks for making this plugin, this would be a really nice feature to add to wordpress core for sure :)

### CODE REVIEW

**Comment by danieltj27** [comment]
Thanks for submitting this, I’ll take a proper look soon. 🙂

File: darkmode.scss [hunk 1]
```diff
@@ -44,6 +44,7 @@ $ultra-grey:      #191f25;
 $dark-silver:     #50626f;
 $base-blue:       #2e74aa;
 $light-blue:      #4092d2;
+$focus-blue:      #5b9dd9;
```
Comment (danieltj27) [inline darkmode.scss]: Let's try and keep to the colour palette we already have. I think one of the existing colours should work nicely with this.

Comment (loumray) [reply darkmode.scss]: Good Idea. Now using light-blue instead.

**Comment by loumray** [comment]
resolved conflicts with the latest update and did the color change suggested

**Comment by danieltj27** [comment]
Thanks for the update, I'll take a proper look at this soon.

**Comment by danieltj27** [comment]
Thanks for your changes, I've cleaned up a couple of bits in the CSS but everything looked good.

👍

**Comment by loumray** [comment]
Great! Glad that it turns out to be useful. Looking forward to contribute again.

Thank you Dan! Keep up the good work.

**Pull request merged**